Calico/Vpp docs cleanup (#272)
tigera/docs: calico/getting-started/kubernetes/vpp/getting-started.mdx

**Legacy options**

> We maintain legacy support for the `CALICOVPP_INTERFACE` and `CALICOVPP_NATIVE_DRIVER` env vars that corresponds to the following:

Reviewer: ```suggestion
We maintain legacy support for the `CALICOVPP_INTERFACE` and `CALICOVPP_NATIVE_DRIVER` environment variables:
```

Author: Done. Thanks for catching it.

> `CALICOVPP_INTERFACE` -> `uplinkInterfaces[0].interfaceName`

Reviewer: Nice catch! Thanks!

Author: Thanks!
